solr/core/src/java/org/apache/solr/core/CoreContainer.java

```diff
@@ -1788,6 +1786,7 @@ public void unload(String name, boolean deleteIndexDir, boolean deleteDataDir, b
   }
 
   public void rename(String name, String toName) {
+    apiAssumeStandalone();
```

Comment:
change java doc reflecting this change

Reply:
I think it'd be more pertinent to update the Solr Ref Guide, which is what users would see.  This method here is very internal.  At least if a user tries it, it'll now not work with an error telling them so, as opposed to before, bad things would probably happen.